### ISSUE TITLE
Add 'Public Network Range' and 'Internal Network Range' types to 'Systems'

### ISSUE THREAD

**Issue by febrezo** [cyb3rfox/Aurora-Incident-Response#61]
In major incidents, keeping track of the public and internal network ranges is critical to track what kind of systems we are working to. Currently there is no space to keep track of it in Aurora. My proposal is to include two new types under the 'Systems' tab.

**Comment by cyb3rfox** [comment]
Makes sense and sounds like a way to store it. we put information like that under case notes usually.

**Comment by sidomir7** [comment]
I would love it too.
